src/diffusers/loaders/unet.py

```diff
@@ -561,6 +561,100 @@ def _unfuse_lora_apply(self, module):
             if isinstance(module, BaseTunerLayer):
                 module.unmerge()
 
+    def _expand_lora_scales_dict(
+        self, scales, blocks_with_transformer: Dict[str, int], transformer_per_block: Dict[str, int]
+    ):
+        """
+        Expand input into a weight dict with a weight per transformer
+
+        Parameters:
+            blocks_with_transformer (`Dict[str, int]`):
+                Dict with keys 'up' and 'down', showing which blocks have transformer layers
+            transformer_per_block (`Dict[str, int]`):
+                Dict with keys 'up' and 'down', showing how many transformer layers each block has
+
+        E.g. turns
```

Comment:
I think the example would be more informative if it showed this method being called with certain arguments and the corresponding output.

Reply:
Agree - have expanded the example